test/test_activation.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
+
+import torch
+import unittest
+from torch.nn.modules.activation import MultiheadAttention
+
+class TestActivation(unittest.TestCase):
```

Comment:
This should fit neatly into one of our existing test files

Reply:
I'm moving it to `/test/test_native_mha.py`